core/src/main/java/org/jboss/pnc/build/finder/core/BuildFinder.java

```diff
@@ -1470,7 +1472,11 @@ private static Set<LicenseInfo> addLicensesToBuilds(
                 localArchive.getLicenses().addAll(licenseInfos);
                 allLicenses.addAll(licenseInfos);
             } else {
-                LOGGER.error("No matching archive found for file {}", boldRed(filename));
+                if (notFoundChecksums.values().stream().noneMatch(v -> v.contains(filename))) {
+                    LOGGER.error("No matching archive found for file {}", boldRed(filename));
+                } else {
+                    LOGGER.warn("Checksum not found for {} so can't add to license archive", red(filename));
```

Comment:
May just silently skip these since currently licenses are only stored for found builds.